skills/declarative/calendarSkill/calendarskill.dialog

```diff
@@ -23,20 +23,38 @@
           "name": "calendarSkill.Greeting"
         },
         {
-          "$kind": "Microsoft.BeginDialog",
-          "$designer": {
-            "id": "RvRTf5"
-          },
-          "activityProcessed": true,
-          "dialog": "AuthenticationDialog"
-        },
-        {
-          "$kind": "Microsoft.SendActivity",
+          "$kind": "Microsoft.IfCondition",
           "$designer": {
-            "id": "gKWYRF",
-            "name": "Send a response"
+            "id": "25rJrE"
           },
-          "activity": "${SendActivity_gKWYRF()}"
+          "condition": "= turn.activity.channelId == 'msteams'",
```

Comment:
Does this show the launch button when users install the bot in their chats themselves? Wondering if there's a way to only show this for chats in a Team or chats with multiple people

Reply:
Yes...Let me look into it...

Reply:
Added a if condition to check 'Personal' channel.